bin/i18n-codeorg/crowdin.yaml.erb

```diff
@@ -21,4 +19,5 @@ files:
       - /source/blockly-mooc/applab.json
       - /source/blockly-mooc/calc.json
       - /source/blockly-mooc/eval.json
+      - /source/blockly-mooc/gamelab.json
```

Comment:
TODO: add to wiki how to specify which apps aren't translated. Needs to be updated in sync-codeorg and in crowdin.yaml.